python/lsst/analysis/tools/interfaces/datastore/_dispatcher.py

```diff
@@ -591,6 +592,9 @@ def _prepareBundle(
                             )
                             resultsTrimmed = True
                             continue
+                        # JSON will not serialize np.float32, must cast.
+                        if isinstance(value, np.float32):
```

Comment:
I think `isinstance(value, numbers.Real)` will work, which is what is done elsewhere in analysis_tools and is more general than np.float32 check.

Reply:
That also catches ints, which I was trying to avoid casting unnecessarily.